content/docs/intro/cloud-providers/azure-nextgen/setup.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Azure Next Gen Setup
+meta_desc: This page provides an overview of how to set up and configure credentials
+           for the Pulumi Azure Next Gen Provider.
+aliases: ["/docs/reference/clouds/azure/setup/"]
+---
+
+The [Pulumi Azure Next Gen provider]({{< relref "./" >}}) uses the Azure SDK to manage and provision resources.
+
+> Pulumi relies on the Azure SDK to authenticate requests from your computer to Azure. Your credentials are never sent
+> to pulumi.com.
+
+We recommend using the [Azure CLI 2.0](https://github.com/Azure/azure-cli), instead of the legacy
+[Azure xPlat CLI](https://github.com/Azure/azure-xplat-cli).  We support both, but we recommend upgrading.
+The below instructions assume that you are using the Azure CLI 2.0.
+
+## Credentials
+
+Pulumi can authenticate to Azure using a Service Principal or the Azure CLI.
+
+If you're running the Pulumi CLI locally, in a developer scenario, we recommend using the Azure CLI.  For team
+environments, particularly in CI, a Service Principal is recommended.
+
+> **Note:** Authenticating using the CLI will not work for Service Principal logins (e.g.,
+> `az login --service-principal`).  For such cases, authenticate using the Service Principal method instead.
+
+## CLI Authentication
+
+Simply login to the Azure CLI and Pulumi will automatically use your credentials:
+
+```bash
+$ az login
+To sign in, use a web browser to open the page https://aka.ms/devicelogin and enter the code XXXFAKEXXX to authenticate.
```

Comment:
I believe this is changed and a device code isn't needed anymore. Could you please log in with `az login` and update the prompt?